synthesizer/src/vm/mod.rs

```diff
@@ -332,6 +332,65 @@ function compute:
             .clone()
     }
 
+    pub(crate) fn sample_execution_transaction_with_fee(rng: &mut TestRng) -> Transaction<CurrentNetwork> {
```

Comment:
It's advisable potentially to make this module only available in test so it isn't available to production methods. Also potentially it should be moved to `vm/helpers/mod.rs` or a new file like maybe `vm/helpers/test_utils.rs`?

```suggestion
    #[cfg(test)]
    pub(crate) fn sample_execution_transaction_with_fee(rng: &mut TestRng) -> Transaction<CurrentNetwork> {
```

Reply:
This function exists within the `test_helpers` module in `pub(crate) mod test_helpers {`, so it shouldn't be accessible from outside of the test context

Reply:
Ah yes I see, my bad. Perhaps moving to helpers is still an apt choice? Either way not a blocker.